insights/tests/parsers/test_cpuset_cpus.py

```diff
@@ -1,13 +1,43 @@
 from insights.parsers import cpuset_cpus
 from insights.tests import context_wrap
-
+import doctest
```

Comment:
Could you please move this above the absolute imports.

```
import doctest

from insights.parsers import cpuset_cpus
from insights.tests import context_wrap
```

Reply:
@ryan-blakley - Updated, please check

Reply:
Thank you.